README.md

```diff
@@ -1,9 +1,25 @@
-# openinput-cm3
+# openinput-mouse
 
-Open source mouse firmware based on [libopencm3](https://github.com/libopencm3/libopencm3).
+Open source mouse firmware.
+
+```
+.
+├── tools	# Utilities
+├── Targets	# Meson cross-compile files
```

Comment:
Lowercase, and please rename it to `build-targets`.

Reply:
ups, i always use lower case.